spec/datadog/tracing/contrib/grpc/datadog_interceptor/client_spec.rb

```diff
@@ -31,9 +31,10 @@
   end
 
   context 'using client-specific configurations' do
+    let(:deadline) { Time.now }
```

Comment:
I recommend keeping a predictable time for the test suite, something like:
```suggestion
    let(:deadline) { Time.utc(2022, 1, 2, 3, 4, 5, 678901) }
```
For this example, `deadline#to_f` will return `1641092645.678901`, which has the maximum precision supported.

This allows us to test that we received exactly `1641092645678` here:
```ruby
       expect(span.get_tag(Datadog::Tracing::Contrib::GRPC::Ext::TAG_CLIENT_DEADLINE)).to eq(1641092645678)
```